index.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portifolio</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body> <!-- Header start-->
+    <header>
+        <div class="nav-container">
+            <a href="#" class="logo">WELCOME</a>
+            <nav>
+                <button class="menu" type="button" >
+                    <img src="images/menu.gif" alt="a menu link">
+                </button>
+            </nav>
+        </div>
+    </header>   <!-- Header end-->
+
+     <!-- main start-->
+    <section class="main">
+        <div class="intro">
+            <h1>Hey there. I’m Yordi ! </h1>
+            <h1><span class="a-span">I’m a software developer</span></h1>
```

Comment:
- [ ] h1 tag is only used once on a webpage and it semantically determines the core of the page title. every other h should be from h2 to h6. In that sense, I would recommend you change the h on line 26.